modules/dxp/apps/saml/saml-impl/src/main/java/com/liferay/saml/util/internal/SamlSendAuthnRequestUtilImpl.java

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation: